src/js/detektorCirilice.js

```diff
@@ -0,0 +1,21 @@
+/* eslint-disable no-unused-vars */
+const poljeTeksta = document.getElementById('poljeTeksta')
+const validacija = document.getElementById('validacija')
+const tasterZaValidaciju = document.getElementById('tasterZaValidaciju')
+const latinica = 'A,a,B,b,C,c,Č,č,Ć,ć,D,d,Dž,dž,Đ,đ,E,e,F,f,G,g,H,h,I,i,J,j,K,k,L,l,Lj,lj,M,m,N,n,Nj,nj,O,o,P,p,R,r,S,s,Š,š,T,t,U,u,V,v,Z,z,Ž,ž'.split(',')
+const cirilica = 'А,а,Б,б,Ц,ц,Ч,ч,Ћ,ћ,Д,д,Џ,џ,Ђ,ђ,Е,е,Ф,ф,Г,г,Х,х,И,и,Ј,ј,К,к,Л,л,Љ,љ,М,м,Н,н,Њ,њ,О,о,П,п,Р,р,С,с,Ш,ш,Т,т,У,у,В,в,З,з,Ж,ж'.split(',')
+
+function validirajCirilicu () {
+  for (let i of poljeTeksta.value.split("")) {
+    if (cirilica.includes(i)) {
+      validacija.innerHTML = 'Tekst je napisan u ćirilici'
+      break
+    } else if (latinica.includes(i)) {
+      validacija.innerHTML = 'Tekst je napisan u latinici'
+    } else {
+      validacija.innerHTML = 'Piši srpski da te ceo svet razume'
+    }
+  }
+}
+
```

Comment:
Ovo bi trebalo da se preradi da vraća vrednost sa return.
Šta radi break u 12 liniji i zašto postoji u tom bloku a u bloku sa latinicom ne?